CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## [Unreleased]
 
+### Fixed
+- Crash in commands that ask for user input (like `pcs cluster destroy`) when
+  stdin is closed ([ghissue#612])
+
+[ghissue#612]: https://github.com/ClusterLabs/pcs/issues/612
+
+
+## [0.11.5] - 2023-03-01
```

Comment:
I guess this should not be here

Reply:
Definitely not, not sure how I missed it. I'm going to fix it.